### ISSUE TITLE
Cannot reconnect Australia server / ranked

### ISSUE THREAD

**Issue by commentator24** [ValveSoftware/Dota2-Gameplay#7468]
was playing BH, vs OD mid. suddenly lag on first minute after I try to toss shuriken? can't rejoin. it's over 5 minutes already yet still stuck.
![image](https://user-images.githubusercontent.com/109324155/224485269-e916c88f-e8ea-4ab8-a467-b1a7d3af9094.png)

https://user-images.githubusercontent.com/109324155/224485280-1b11828c-985d-4560-a5cb-ebb5b9634858.mp4

**Comment by CHIPPYPOP** [comment]
I’ve had the same issue. Ranked Aus servers. Still stuck in a reconnect loop ~15 mins later. I was Treant pos4 laning with tusk pos 3 against spec/pudge. 
![Uploading image.jpg…]()

**Comment by commentator24** [comment]
ok could leave now, but after i found a new game, the same thing happened :( 
this time during first drafting phase, just kicked everyone to main menu
![image](https://user-images.githubusercontent.com/109324155/224486289-c462103a-ea7d-47c8-b213-b894f3142ba2.png)

**Comment by commentator24** [comment]
err now disabled until thursday im sad now 😭 😭 
![image](https://user-images.githubusercontent.com/109324155/224487192-5fb0d2df-8b3e-40d8-b9f4-a0c419c026bd.png)

this must be an error @bwvalve 
my id 48546538

**Comment by commentator24** [comment]
seems i was able to queue again after i restart dota. finger crossed doesnt happen again

**Comment by commentator24** [comment]
not sure if this was because of unstable server 5 days ago
or because I disconnected before game about to end (we lost teamfight, enemy about to push throne, we already lost 2 rax). i said to team i need to poop, u can pause or let them end.
or because treant reported me while i reported him.

![image](https://user-images.githubusercontent.com/109324155/225742761-7ee19d05-83a5-468e-8c57-71f5143c975e.png)
after i came back from my poop break i had 30 min penalty 😢 can I get penalty if I disconnected 3-4 min before the game ends? sigh
